tools-local/unix/invoke-extension.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "usage: $0 <script-name> <repository-root> <tool-name> <override-scripts-folder-path> [strict-tool-version-match] [tool-path]"
```

Comment:
Ok. All args are required. Updated as -
    echo "usage: $0 <script-name> <repository-root> <tool-name> <override-scripts-folder-path> <strict-tool-version-match> <tool-path>"


---
In reply to: [110417927](https://github.com/ravimeda/corefx/pull/9#discussion_r110417927) [](ancestors = 110417927)